src/common.rs

```diff
@@ -6,6 +6,8 @@ pub(crate) mod frequency_sketch;
 pub(crate) mod thread_pool;
 pub(crate) mod unsafe_weak_pointer;
 
+pub(crate) type Weighter<K, V> = Box<dyn Fn(&K, &V) -> u64 + Send + Sync + 'static>;
```

Comment:
In caffeine this is `Weigher`. I think this is a typo?

Reply:
Yes, it is a typo. Thank you for letting me know.